hack/build-platform.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Build all compile targets and the base binaries for only the platform of the
+# host it's building on. (i.e. - no cross compile)
+STARTTIME=$(date +%s)
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+
+source <(go env)
+platforms=( "${GOHOSTOS}/${GOARCH}" )
+if [[ -n "${OS_ONLY_BUILD_PLATFORMS-}" ]]; then
+  filtered=()
+  for platform in ${platforms[@]}; do
+    if [[ "${platform}" =~ "${OS_ONLY_BUILD_PLATFORMS}" ]]; then
+      filtered+=("${platform}")
+    fi
+  done
+  platforms=("${filtered[@]}")
+  echo "DEBUGGING: ${filtered[@]}"
+fi
+
+# Build the primary client/server for the host platform
+OS_BUILD_PLATFORMS=("${platforms[@]}")
+if [[ "${GOHOSTOS}/${GOARCH}" == "linux/amd64" ]]; then
```

Comment:
Yeah, sorry ... I code primarily in python so long-strings of if statements is just a habit. I'll update.